constants.py

```diff
@@ -1,7 +1,18 @@
-class window:
+class mainWindow:
     width, height = 800, 600
     title = "EVSTimulator"
-    background = "white"
+
+class simulationWindow:
+    width, height = mainWindow.width * 2 // 3, mainWindow.height * 4 // 5 
+
+class buttonsWindow:
+    width, height = mainWindow.width * 4 // 5, mainWindow.height * 1 // 5
+
+class textWindow:
+    width, height = mainWindow.width * 1 // 5, mainWindow.height * 1 // 5
+
+class plotWindow:
+    width, height = mainWindow.width * 1 // 3, mainWindow.height * 4 // 5
 
```

Comment:
I have created these classes like that. Maybe there is a better way to configure these values. Do you know any?

Also, there is a bug, which somewhat mismatch in the sizes of the width. If you increase the width size of the main window while running, you'll see some more space is coloured.